pyfftw/interfaces/_utils.py

```diff
@@ -90,6 +89,11 @@ def _Xfftn(a, s, axes, overwrite_input, planner_effort,
             # the check and the lookup
             FFTW_object = None
 
+    if reload_after_transform:
+        # Instead of reloading after transform, we just transform a
+        # copy instead.
+        a = a.copy(order='K')  # Match original layout as closely as possible
```

Comment:
Note that in the two special cases where FFTW will always clobber the original array, we instead transform a copy.  In this particular case, the copy will also be used for planning purposes so there is no issue with striding.